VARIABLES.md

```diff
@@ -566,9 +566,9 @@ Default:  "{{ [ groups['kafka_broker'] | default(['localhost']) | length, 3 ] |
 
 ### kafka_broker_metrics_reporter_enabled
 
-Boolean to enable the metrics reporter
+Boolean to enable the metrics reporter. Defaults to true if Control Center in inventory. Enable if you wish to have monitoring inceptors to report to a centralized monitoring cluster.
```

Comment:
```suggestion
Boolean to enable the metrics reporter. Defaults to true if Control Center in inventory. Enable if you wish to have monitoring interceptors to report to a centralized monitoring cluster.
```

Reply:
This one should probably say metrics reporter not monitoring interceptors